Makefile

```diff
@@ -54,10 +54,11 @@ install-latest-release:
 	}
 
 
```

Comment:
it might be better to change the github workflow instead of making lint do double duty here. What are your thoughts @leg100 ?